.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Please add all files you want to ignore in .gitignore in root.
Please add /.idea to gitignore.